docs/integrations/sources/file.md

```diff
@@ -51,14 +51,20 @@ This source produces a single table for the target file as it replicates only on
 
 **This connector does not support syncing unstructured data files such as raw text, audio, or videos.**
 
-## Getting Started (Airbyte Cloud)
+## Getting Started
+
+<!-- env:cloud -->
+**For Airbyte Cloud:**
 
 Setup through Airbyte Cloud will be exactly the same as the open-source setup, except for the fact that local files are disabled.
+<!-- /env:cloud -->
 
-## Getting Started (Airbyte Open-Source)
```

Comment:
Wouldn’t be enough to add `<!-- env:oss —>` before this line?
or do we want to change the titles?

Reply:
I improved consistency with the other connectors pages again